commons-vfs2/src/test/java/org/apache/commons/vfs2/NamingTests.java

```diff
@@ -404,7 +404,7 @@ public void testRootFileName() throws Exception {
         // Test that the root path is "/"
         assertEquals("root path", "/", rootName.getPath());
 
-        // Test that the root basname is ""
+        // Test that the root basename is ""
```

Comment:
"basename" is not a word either, it should be "base name".

Reply:
Done.